packages/ui/components/calendar/src/LionCalendar.js

```diff
@@ -362,10 +367,8 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
    */
   __calculateInitialCentralDate() {
     if (this.centralDate === this.__today && this.selectedDate) {
-      // initialised with selectedDate only if user didn't provide another one
+      // initialized with selectedDate only if user didn't provide another one
       this.centralDate = this.selectedDate;
-    } else {
```

Comment:
@gerjanvangeest I guess you meant to replace this logic with `this.findNearestEnabledDate`, instead of removing it. I created the PR that does it: https://github.com/ing-bank/lion/pull/2361